docs/development/listeners.md

```diff
@@ -66,3 +66,24 @@ is connection oriented. It's another case for `DMAP`, since that protocol is
 request based. For now, this interface is implemented by the push updates
 API (to be clear: for `DMAP`). So when the push updates API failes to establish
 a connection, the callbacks in this interface will be called.
+
+## Power State Updates
+
+It is possible to get callbacks whenever a device power state is changed, 
+e.g. the device turned on or turned off. The API is defined by the
+`interface.PowerListener` interface and works similarly to how push updates works.
+
+Here is a simple example:
+
+```python
+class PowerListener:
+
+    def powerstate_update(self, old_state, new_state):
+        print('Power state changed from %s to %s', old_state, new_state)
```

Comment:
C-style is only supported by `logging`, here you should use `.format`. When I decide to not support python 3.5 anymore, we can use f-strings instead.